src/App.tsx

```diff
@@ -1,9 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [movieName, setMovieName] = useState('');
```

Comment:
```suggestion
  const [movieName, setMovieName] = useState('');
```
It is confusing var names, if it is a `query`, so let's use this meaningful name for state name